config/routes.rb

```diff
@@ -6,6 +6,7 @@
   namespace :api do
     namespace :v1 do
       resources :books, only: [:index, :show]
+      get 'books/search/:isbn', to: 'books#search'
```

Comment:
Hum, I prefer you pass it a query params and not path params. We prefer reserve path params for id in our current DB